core/src/subgraph/instance_manager.rs

```diff
@@ -595,6 +596,30 @@ where
     }
 }
 
+#[derive(thiserror::Error, Debug)]
+enum ProcessBlockError {
```

Comment:
To me, `BlockProcessingError` or even `IndexingError` would be clearer than `ProcessBlockError`

Reply:
Renamed to `BlockProcessingError`.